zulipterminal/helper.py

```diff
@@ -535,9 +535,18 @@ def match_emoji(emoji: str, text: str) -> bool:
 
 
 def match_topics(topic_names: List[str], search_text: str) -> List[str]:
-    return [
-        name for name in topic_names if name.lower().startswith(search_text.lower())
-    ]
+    res = []
+    delimiters = "-_/"
+    trans = str.maketrans(delimiters, len(delimiters) * " ")
+    for topic_name_raw in topic_names:
```

Comment:
Again minor re naming:
- raw? `full_topic_name`? `complete_topic_name`? `entire_topic_name`?
- topic_name_lst? Sure it's a list, and we could add an `i`, but it's not a list of topic names... or is the `lst` suggesting something else?